pythonmarketo/client.py

```diff
@@ -236,7 +237,29 @@ def create_lead(self, lookupField, lookupValue, values):
             ]
         }
         return self.post(data)
+    
```

Comment:
Do you have any test for this method so that I can run it before merging the pulled changes?